domain/src/test/java/starterproject/domain/TransmissionTest.java

```diff
@@ -0,0 +1,5 @@
+package starterproject.domain;
+
+class TransmissionTest {
```

Comment:
Remove this class, no need for it at the moment and it doesn't define a useful framework for us (or another programmer) to use later.

 In an ideal world, every class should be tested so, with that in mind, just defining that we need a test is kind of redundant. 

If there was some non-trivial functionality or non-trivial setup ready for a test then maybe we have a case for having it.